src/cascadia/TerminalSettingsEditor/Interaction.xaml

```diff
@@ -28,64 +28,64 @@
         <StackPanel Style="{StaticResource SettingsStackStyle}">
             <!--  Copy On Select  -->
             <local:SettingContainer x:Uid="Globals_CopyOnSelect">
-                <ToggleSwitch IsOn="{x:Bind State.Globals.CopyOnSelect, Mode=TwoWay}"
+                <ToggleSwitch IsOn="{x:Bind ViewModel.GlobalSettings.CopyOnSelect, Mode=TwoWay}"

```

Comment:
Shouldn't the viewmodel abstract away the need to access the actual model?

Reply:
Fixed!